README.md

```diff
@@ -1,55 +1,61 @@
 
-1. Install the tooling by running the following command after navigating to the appropriate folder:
 ```bash
+# Navigate to the appropriate folder
 $ cd some/folder/flowcrypt-browser
+# Install the tooling
 $ npm install
 ```
+
 2. To build the project (currently available for Linux and Mac only), run the following command:
 
 ```bash
 $ npm run build
 ```
-After running this command, you can find your built project in the `build/chrome-consumer` and `build/firefox-consumer` folders.
+
+After executing the build command, you can find your built project in the `build/chrome-consumer` and `build/firefox-consumer` folders.
+
+> Note: The `build` folder also contains other versions of the browser extension.
 
 To load the extension in Google Chrome, please follow these steps:
 
 1. Open your Chrome browser and navigate to `chrome://extensions/`.
-2. If not already enabled, toggle on the `Developer mode`    switch located at the top-right corner.
-3. Click on the `Load Unpacked` button.
-4. Browse to and select the appropriate folder, either `build/chrome-consumer` or `chrome-enterprise`.
+2. If it isn't already enabled, toggle on the **Developer mode**. The switch button is located at the top-right corner.
+3. Click on the `Load unpacked` button.
+4. Browse to the `flowcrypt-browser/build` folder and select the appropriate project version, either `build/chrome-consumer` or `chrome-enterprise`.
 
-Similarly, to load the extension in Firefox, follow these steps:
+Similarly, to load the extension in Firefox, please follow these steps:
 
 1. Open your Firefox browser and navigate to `about:debugging`.
 2. Click on the `This Firefox` tab.
 3. Click on the `Load Temporary Add-on` button located at the top-right corner.
-4. Browse to and select the appropriate folder, either `build/firefox-consumer` or `firefox-enterprise`, and select the `manifest.json` file.
+4. Browse to the `flowcrypt-browser/build` folder and select the appropriate project version, either `build/firefox-consumer` or `firefox-enterprise`, and select the `manifest.json` file.
 
-If you prefer, you can also use the `run_firefox` script (`npm run run_firefox`) included in the package.json file to run the Firefox extension in a separate instance without interfering with the production extension installed in your browser.
+If you wish, you can also use the `run_firefox` script (`npm run run_firefox`) included in the `package.json` file to run the Firefox extension in a separate instance without interfering with the production extension installed in your browser.
 
-Printing debug data to test logs can be done using special `Debug` class:
-https://github.com/FlowCrypt/flowcrypt-browser/tree/master/extension/js/common/platform/debug.ts#L7
+Printing debug data to test logs can be accomplished using a special `Debug` class: [github.com/FlowCrypt/flowcrypt-browser](https://github.com/FlowCrypt/flowcrypt-browser/tree/master/extension/js/common/platform/debug.ts#L7)
```

Comment:
let's make `special Debug class` a link, so it'll look like this:

Printing debug data to test logs can be accomplished using a [special Debug class](https://github.com/FlowCrypt/flowcrypt-browser/tree/master/extension/js/common/platform/debug.ts#L7)